File: app/src/main/java/io/neoterm/view/eks/ControlButton.java
package io.neoterm.view.eks;

import android.view.View;

import io.neoterm.view.ExtraKeysView;

/**
 * @author kiva
 */

public class ControlButton extends ExtraButton {
    public ControlButton(String text) {
        buttonText = text;
    }

    @Override
    public void onClick(View view) {
        ExtraKeysView.sendKey(view, buttonText);
    }
}

File: app/src/main/java/io/neoterm/view/eks/ControlButton.kt
package io.neoterm.view.eks

import android.view.View

/**
 * @author kiva
 */

open class ControlButton(text: String) : ExtraButton() {
    init {
        buttonText = text
    }

    override fun onClick(view: View) {
        ExtraButton.Companion.sendKey(view, buttonText!!)
    }
}
